common/changes/@microsoft/rush/reconfigure-watch_2021-02-13-00-01.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Replace \"--watch\" with a \"watchForChanges: true\" setting in command-line.json, since running in watch mode is not automatically compatible with downstream file watchers, and use of such is the expected common scenario.",
```

```suggestion
      "comment": "Convert the experimental \"--watch\" parameter into a \"watchForChanges: true\" setting in command-line.json, based on user feedback",
```

The detailed rationale can go in the PR notes.